modules/common/condition/funcs_test.go

```diff
@@ -277,6 +278,88 @@ func TestMarkMethods(t *testing.T) {
 	g.Expect(conditions.Get("a")).To(haveSameStateOf(unknownA))
 }
 
+func TestSortByLastTransitionTime(t *testing.T) {
+	g := NewWithT(t)
+
+	time1 := metav1.NewTime(time.Date(2020, time.August, 9, 10, 0, 0, 0, time.UTC))
+	time2 := metav1.NewTime(time.Date(2020, time.August, 10, 10, 0, 0, 0, time.UTC))
+	time3 := metav1.NewTime(time.Date(2020, time.August, 11, 10, 0, 0, 0, time.UTC))
+
+	falseA.LastTransitionTime = time1
+	falseB.LastTransitionTime = time2
+	falseError.LastTransitionTime = time3
+
+	g.Expect(lessLastTransitionTime(falseA, falseB)).To(BeFalse())
+	g.Expect(lessLastTransitionTime(falseB, falseA)).To(BeTrue())
+
+	conditions := Conditions{}
+	cl := CreateList(falseB, falseError, falseA)
+	conditions.Init(&cl)
+	conditions.SortByLastTransitionTime()
+
+	// unknownReady has the current time stamp, do is first
+	g.Expect(conditions).To(haveSameConditionsOf(CreateList(unknownReady, falseError, falseB, falseA)))
+}
+
+func TestMirror(t *testing.T) {
+	g := NewWithT(t)
+
+	time1 := metav1.NewTime(time.Date(2020, time.August, 9, 10, 0, 0, 0, time.UTC))
+	time2 := metav1.NewTime(time.Date(2020, time.August, 10, 10, 0, 0, 0, time.UTC))
+	time3 := metav1.NewTime(time.Date(2020, time.August, 11, 10, 0, 0, 0, time.UTC))
+
+	trueA.LastTransitionTime = time1
+	falseB.LastTransitionTime = time2
+	falseError.LastTransitionTime = time3
```

Comment:
`falseError` is modified here and never used below.  Should this be `falseBError` instead?

Reply:
falseError and  time3 can be removed. I changed the tests to use falseBError and missed to remove the falseError here.